onadata/apps/logger/models/instance.py

```diff
@@ -441,15 +442,17 @@ def get_notes(self):
         # pylint: disable=no-member
         return [note.get_data() for note in self.notes.all()]
 
-    def get_review_status_and_comment(self):
+    def get_review_details(self):
```

Comment:
I tend to be hesitant in renaming functions that have been around for a long time, in this case about 3 years. I would perhaps recommend a new method:

```python
def get_latest_review(self):
    try:
        return self.reviews.latest('date_modified')
    except SubmissionReview.DoesNotExist:
        return None
````
Which then we can use as
```python
review = self.get_latest_review()
if review:
    doc[REVIEW_STATUS] = review.status
    doc[REVIEW_COMMENT] = review.comment
    doc[REVIEW_DATE] = review.date_created.strftime(MONGO_STRFTIME)
```
I think this approach is much cleaner and clearer.

Reply:
It also means you do not end up having to rename old functions, you could mark the old function deprecated so that we eventually change its reference everywhere.